packages/amazonq/.changes/next-release/Feature-a007f02d-d737-4429-9b7d-245aa30d879c.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Enable Free Tier Chat for IAM users & Fix code completion for Sagemaker Pro Tier users"
```

Comment:
Split this into 2 changelog items, eg:
```
{
	"type": "Feature",
	"description": "Chat: Enable free tier for IAM users in Sagemaker."
}
```
and (this second one may need to be more descriptive)
```
{
	"type": "Bug Fix",
	"description": "Inline: Code completion not working for Sagemaker Pro Tier users."
}
```